website/source/intro/getting-started/join.html.md

```diff
@@ -81,7 +81,7 @@ All together, these settings yield a
 ```text
 vagrant@n1:~$ consul agent -server -bootstrap-expect=1 \
 	-data-dir=/tmp/consul -node=agent-one -bind=172.20.20.10 \
-	-config-dir=/etc/consul.d
+	-check-enable-exec=true -config-dir=/etc/consul.d
```

Comment:
Yeah we add a script check on the following example so it's needed to make that work. Same for the other one.